merkletree/pad.go

```diff
@@ -122,6 +122,7 @@ func (pad *PAD) GetSTR(epoch uint64) *SignedTreeRoot {
 func (pad *PAD) TB(key string, value []byte) (*TemporaryBinding, error) {
 	str := pad.latestSTR
 	index, _ := pad.computePrivateIndex(key, pad.policies.vrfPrivate())
+
```

Comment:
Yeah. That's why I said that this branch should be merged after `vrf_integration` branch. Sorry for the noise :(